cabin-sorting/tsconfig.json

```diff
@@ -74,7 +74,7 @@
     // "preserveValueImports": true,                     /* Preserve unused imported values in the JavaScript output that would otherwise be removed. */
 
     /* Interop Constraints */
-    "isolatedModules": true,                          /* Ensure that each file can be safely transpiled without relying on other imports. */
+    // "isolatedModules": false,                          /* Ensure that each file can be safely transpiled without relying on other imports. */
```

Comment:
why?

Reply:
For the outDir one, we kept getting an error because it said the output files would override the src files so by making the the output files be sourced to dict instead, the error disappeared. Mike said this is cause typescripts weird and even though we don’t even have any output files we still need it